ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountStatsMySqlStore.java

```diff
@@ -166,14 +164,23 @@ public AccountStatsMySqlStore(AccountStatsMySqlConfig config, DataSource dataSou
     partitionClassReportsDao = new PartitionClassReportsDao(dataSource, mySqlMetrics);
     this.hostnameHelper = hostnameHelper;
     storeMetrics = new AccountStatsMySqlStore.Metrics(registry);
-    if (!Strings.isNullOrEmpty(localBackupFilePath)) {
-      // load backup file and this backup is the previous stats
-      ObjectMapper objectMapper = new ObjectMapper();
+    if (config.readPreviousStatsFromMysqlDb) {
```

Comment:
Is there a situation where this config should be disabled? If this is not a dangerous feature to roll out and we want to use it in all circumstances, I would prefer not having another config for this.

Reply:
Well, previously I'm planning to remove the config once it has been tested. But you are right, it's not a dangerous since if it's failed we can read from local backup file so I just removed the config.